src/Tizen.Multimedia.MediaPlayer/Player/EqualizerBand.cs

```diff
@@ -29,23 +29,19 @@ public class EqualizerBand
     {
         private readonly AudioEffect _owner;
         private readonly int _index;
+        private readonly int _frequency;
+        private readonly int _frequencyRange;
```

Comment:
Why need it as filed member?
nobody use it, except constructor
I think, it should be a local variable

Reply:
i'll fix it.